lib/ZwaveClient.js

```diff
@@ -1838,8 +1838,8 @@ ZwaveClient.prototype.getInfo = function () {
   return info
 }
 
-ZwaveClient.prototype.refreshValues = function () {
-  this.nodes.refreshValues()
+ZwaveClient.prototype.refreshValues = function (nodeId) {
+  this.getNode(nodeId).refreshValues()
```

Comment:
That method returns a Promise which could (theoretically) be rejected - should probably add a .catch()

Reply:
Done in new commit